src/repositories/soft-crud.repository.base.ts

```diff
@@ -168,6 +168,34 @@ export abstract class SoftCrudRepository<
     );
   }
 
+  /**
+   * Method to perform undo the soft delete by Id.
+   * @param id
+   * @param options
+   */
+  async undoSoftDeleteById(id: ID, options?: Options): Promise<void> {
+    const user = await this.findByIdIncludeSoftDelete(id);
```

Comment:
just the variable name can be something better than user
as this is a generic entity it can be either entity or record or something else
but not user

Reply:
yes ma'am. I agree. Will change it.